_bibliography/references.bib

```diff
@@ -61,6 +61,13 @@ @String { tsg
 @String { uai         = {Conference on Uncertainty in Artificial Intelligence (UAI)} }
 @String { wafr         = {Workshop on the Algorithmic Foundations of Robotics} }
 
+@article{palafox2023learning,
```

Comment:
Leave this off until it is accepted, so that the "Publications" tab is only published papers.